x-pack/plugins/observability/server/services/slo/get_burn_rates.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsClientContract } from '@kbn/core-saved-objects-api-server';
+import { ElasticsearchClient } from '@kbn/core-elasticsearch-server';
+import { KibanaSavedObjectsSLORepository } from './slo_repository';
+import { DefaultSLIClient } from './sli_client';
+import { Duration, toDurationUnit } from '../../domain/models';
+import { computeSLI, computeBurnRate } from '../../domain/services';
+
+interface Services {
+  soClient: SavedObjectsClientContract;
+  esClient: ElasticsearchClient;
+}
+
+interface LookbackWindow {
+  name: string;
+  duration: { value: number; unit: string };
```

Comment:
If you use the type `durationType` from the schema, this duration becomes a `Duration` type.